crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -66,18 +72,40 @@ impl UpChainCommand {
 			cli,
 		)
 		.await?;
-		let para_id = self.resolve_parachain_id(&chain, cli).await?;
+
+		let proxy = self.resolve_proxied_address(&chain, cli)?;
+		let para_id = self.resolve_parachain_id(&chain, &proxy, cli).await?;
 		let (genesis_state, genesis_code) = self.resolve_genesis_files(para_id, cli).await?;
-		Ok(UpChain { id: para_id, genesis_state, genesis_code, chain })
+		Ok(UpChain { id: para_id, genesis_state, genesis_code, chain, proxy })
+	}
+
+	// Retrieves the proxied address, prompting the user if none is specified.
+	fn resolve_proxied_address(&self, chain: &Chain, cli: &mut impl Cli) -> Result<ProxyConfig> {
+		if let Some(addr) = &self.proxied_address {
+			return Ok(ProxyConfig::Address(addr.clone()));
+		}
+		if cli.confirm("Would you like to use a proxy for registration? This is considered a best practice.").interact()? {
+			cli.info("Enter the account the proxy will represent.")?;
```

Comment:
Is this a little ambigous to someone that doesnt know what is required here? The description should also align with that of the command.

The proxy dispatchable uses ` - real: The account that the proxy will make a call on behalf of.`

Reply:
https://github.com/r0gue-io/pop-cli/pull/427/commits/34b95399b9308df0c38f1fcaea77b71019b3cd91